src/compiler/AST/Typed.elm

```diff
@@ -1,16 +1,21 @@
 module AST.Typed exposing
     ( Expr
     , Expr_(..)
+    , LocatedExpr
     , ProjectFields
+    , getExpr
+    , getType
     , isArgument
     , lambda
     , let_
+    , mapExpr
```

Comment:
We currently don't need to expose this, `AST.Typed` is the only module using this